src/components/apps/dungeon/musicPlayer.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import ReactAudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import './css/musicPlayer.css'
+
+interface MusicTrack {
+  src: string;
+  name: string;
+}
+
+interface MusicPlayerProps {
+  tracks: MusicTrack[];
+}
+
+const MusicPlayer = ({ tracks }: MusicPlayerProps) => {
+  const [audioSrc, setAudioSrc] = useState<string>(tracks[0].src);
+  // const [currentTrackName, setCurrentTrackName] = useState<string>(tracks[0].name);
```

Comment:
how come these are commented out in this MR? Something that will be addressed when the music player is finished?

Reply:
Yes. This is for the actual track that will be running. Right now we are not showing but perhaps in the future this might be the case so keeping it commented.